Add nested tabs feature

mikeslim7 opened the pull request:
Changed the plugin to enable nested tabs. Tested with vertical tabs nested in horizontal tabs

mikeslim7 committed:
Update easyResponsiveTabs.js

Updated the modified js that supports nested tabs

mikeslim7 committed:
Update Index.html

Updated the index.html file to include the nested tables and they js.

mikeslim7 committed:
Update README.md

Updating Brand README.md

mikeslim7 committed:
Update README.md

Adding Support of nested tabs

Commit:
bug fixing for accordion nested tabs

Commit:
bug fixing for accordion nested tabs

mikeslim7 committed:
Delete vcs.xml

mikeslim7 committed:
Delete modules.xml

mikeslim7 committed:
Delete misc.xml

mikeslim7 committed:
Delete fileColors.xml

mikeslim7 committed:
Delete encodings.xml

mikeslim7 committed:
Delete Easy-Responsive-Tabs-to-Accordion.iml

mikeslim7 committed:
Delete scope_settings.xml

Cammoy commented:
Nested tabs does not seem to work on horizontal layouts because its calling the same id as the parent tab.

Could you explain how the nested tab feature is achieved please?

mikeslim7 committed:
Bug fix for accordion and addition of "slidespeed" 

- Bug fixed for non-responding action on the 1st accordion tab activation.
- Added support for changing slide speed at initialization using "slidespeed"

mikeslim7 committed:
Adding slidespeed option

mikeslim7 committed:
update

mikeslim7 committed:
update

mikeslim7 committed:
rename options var

mikeslim7 committed:
update

mikeslim7 committed:
replaceState correction
